src/Camera.android.tsx

```diff
@@ -13,7 +12,8 @@ const Camera = React.forwardRef<CameraApi, CameraProps>((props, ref) => {
     capture: async (options = {}) => {
       // Because RN doesn't support return types for ViewManager methods
       // we must use the general module and tell it what View it's supposed to be using
-      return await RNCameraKitModule.capture(options, findNodeHandle(nativeRef.current ?? null));
+      // @ts-ignore make TS happy
+      return await NativeCameraKitModule.capture(options, findNodeHandle(nativeRef.current ?? null));
```

Comment:
What's the problem here?

Reply:
Wouldn't `nativeRef.current ?? undefined` work (or simply `nativeRef.current` depending on its type)?

Reply:
It was like this before, not sure why. I can change it though 🤷‍♂️

Reply:
Should be fixes with 79dfe341899ebcf39e412973cd26538504d370d4